### PR TITLE
Feat: Modified search functionality

### PR DESCRIPTION
**Pull request by vinc3nati**
- Search with categories and names
image

image

### CODE REVIEW

**Review by PeeyushPrashant — APPROVED** [review]
looks good to merge

File: src/components/navbar/navbar.css [hunk 1]
```diff
@@ -4,16 +4,46 @@
   left: 0;
 }
 
+.search-bar {
+  position: relative;
+}
+
+.search-list {
+  position: fixed;
+  background: var(--color-tertiary-dark) !important;
```
Comment (OmKakatkar) [inline src/components/navbar/navbar.css:13]: Avoid !important in CSS

File: src/pages/Search/Search.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { useData } from "../../contexts/data-context";
+
+export const Search = () => {
+  const {
+    state: { products },
+  } = useData();
+  const location = useLocation();
+  const from = location.pathname;
+  const navigate = useNavigate();
+  const query = new URLSearchParams(location.search);
```
Comment (OmKakatkar) [inline src/pages/Search/Search.jsx:13]: This is something new for me

**Review by OmKakatkar — APPROVED** [review]
Project Structure is good
Functionality is working as expected
Like the suggestions under the search while typing. Great work

**Pull request merged**